build/templates/__init__.py.mako

```diff
@@ -4,6 +4,7 @@ ${template_parameters['encoding_tag']}
 enums = template_parameters['metadata'].enums
 config = template_parameters['metadata'].config
 module_name = config['module_name']
+registry_name = config['driver_registry'] if 'driver_registry' in config else config['driver_name']
```

Comment:
Please update the PR to mention support for this new metadata field.

Reply:
This is not new. It was used by nise. It is here so that nise does not need a specialized template.

I will update the decsription to mention this.